opm/io/eclipse/EclIOdata.hpp

```diff
@@ -24,26 +24,28 @@
 
 namespace Opm { namespace EclIO {
 
-    // type MESS have no assisiated data 
+    // type MESS have no assisiated data
     enum eclArrType {
-        INTE, REAL, DOUB, CHAR, LOGI, MESS
+        INTE, REAL, DOUB, CHAR, LOGI, MESS, C0NN
```

Comment:
The name should rather be `CNNN` - i.e. `C999` is fully valid.

Reply:
The name is based on the file format manual from the commercial simulator 

 ```C0nn CHARACTER(len=nn) strings (for example C065 for 65-character strings)```

I would prefer to stick with the name from the manual.

Reply:
OK - that makes sense